### DISCUSSION TITLE
Podman/SELinux users: non-fatal SELinux error by immich-server

### DISCUSSION THREAD

**Discussion by bertmelis** [immich-app/immich#2108] [Q&A]
I'm using Rocky Linux 9.1 with Podman and SELinux. Immich runs as a non-privileged user.

The immich-server container gives a lot of SELinux errors:

```
SELinux is preventing /usr/sbin/xtables-nft-multi from ioctl access on the directory /home/podmanager/.local/share/containers/storage/overlay/4650970b5399fe9b02b1560682398348a32100f66b679b18241b121990df4c20/merged.
```

The container itself doesn't show and strange errors. It's not blocking any obvious things in the Immich stack itself. Everything seems to run fine.

Anyone seeing the same behaviour?
Any ideas?

**Comment by andrewdunndev** [comment]
There is a discussion of documenting podman for hosting immich [over here](https://github.com/immich-app/immich/discussions/1912). I didn't see these errors on my system (centos 9 stream).

Couple Questions:
 - Are you running podman as root or as a user?
 - What is the specific podman invocations you're running?
